### PR TITLE
Add Readme and QD for rosidl_default_runtime

### PR DESCRIPTION
**Pull request by brawner**
This provides a current level quality declaration only for the rosidl_default_runtime package as `rosidl_default_generators` is not currently on the list for QDs.

Signed-off-by: Stephen Brawner <brawner@gmail.com>

### CODE REVIEW

File: rosidl_default_runtime/QUALITY_DECLARATION.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+`rosidl_default_runtime` does not contain any c or c++ code, therefore changes will not affect ABI stability.
+
+## Change Control Process [2]
+
+`rosidl_default_runtime` follows the recommended guidelines for ROS Core packages in the [ROS 2 Developer Guide](https://index.ros.org/doc/ros2/Contributing/Developer-Guide/#package-requirements).
+
+### Change Requests [2.i]
+
+This package requires that all changes occur through a pull request.
```
Comment (Blast545) [inline rosidl_default_runtime/QUALITY_DECLARATION.md]:
```suggestion
All changes will occur through a pull request, check [ROS 2 Developer Guide](https://index.ros.org/doc/ros2/Contributing/Developer-Guide/#change-control-process) for additional information.
```

File: rosidl_default_runtime/QUALITY_DECLARATION.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+Currently nightly results can be seen here:
+* [linux-aarch64_release](https://ci.ros2.org/view/nightly/job/nightly_linux-aarch64_release/lastBuild/testReport/rosidl_default_runtime/)
+* [linux_release](https://ci.ros2.org/view/nightly/job/nightly_linux_release/lastBuild/testReport/rosidl_default_runtime/)
+* [mac_osx_release](https://ci.ros2.org/view/nightly/job/nightly_osx_release/lastBuild/testReport/rosidl_default_runtime/)
+* [windows_release](https://ci.ros2.org/view/nightly/job/nightly_win_rel/lastBuild/testReport/rosidl_default_runtime/)
+
+## Vulnerability Disclosure Policy [7.i]
+
+This package does not yet have a Vulnerability Disclosure Policy
```
Comment (Blast545) [inline rosidl_default_runtime/QUALITY_DECLARATION.md:109]: Consider adding a table at the end summarizing the current state of the package

**Review by Blast545 — COMMENTED** [review]
Minor comments, otherwise LGTM

File: rosidl_default_runtime/QUALITY_DECLARATION.md (hunk 1 above)
Comment (brawner) [reply rosidl_default_runtime/QUALITY_DECLARATION.md]: Done

File: rosidl_default_runtime/QUALITY_DECLARATION.md (hunk 2 above)
Comment (brawner) [reply rosidl_default_runtime/QUALITY_DECLARATION.md:109]: I guess I tend to think the issue (https://github.com/ros2/rosidl_defaults/issues/6) makes it pretty clear the current state of the package and what else needs to be done, and the table just adds a third place where the current state of the package needs to be tracked and updated.

**Review by Blast545 — APPROVED** [review]
LGTM!

File: rosidl_default_runtime/QUALITY_DECLARATION.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+
+`rosidl_default_runtime` does not provide any public API of its own.
+
+### API Stability Within a Released ROS Distribution [1.iv]/[1.vi]
+
+`rosidl_default_runtime` does not provide its own API and therefore API stability will not be impacted by changes.
+
+### ABI Stability Within a Released ROS Distribution [1.v]/[1.vi]
+
+`rosidl_default_runtime` does not contain any c or c++ code, therefore changes will not affect ABI stability.
```
Comment (ahcorde) [inline rosidl_default_runtime/QUALITY_DECLARATION.md]:
```suggestion
`rosidl_default_runtime` does not contain any C or C++ code, therefore changes will not affect ABI stability.
```

**Review by ahcorde — APPROVED** [review]
LGTM with a minor fix

File: rosidl_default_runtime/QUALITY_DECLARATION.md (hunk 3 above)
Comment (brawner) [reply rosidl_default_runtime/QUALITY_DECLARATION.md]: Fixed.

**Pull request merged**